templates/spa.php

```diff
@@ -0,0 +1,235 @@
+                        <nav class="space-y-1">
+                            <a href="#user" class="bg-gray-50 text-indigo-700 hover:text-indigo-700 hover:bg-white group rounded-md px-3 py-2 flex items-center text-sm font-medium" aria-current="page">
+                                <svg class="text-indigo-500 group-hover:text-indigo-500 flex-shrink-0 -ml-1 mr-3 h-6 w-6" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor" aria-hidden="true">
+                                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5.121 17.804A13.937 13.937 0 0112 16c2.5 0 4.847.655 6.879 1.804M15 10a3 3 0 11-6 0 3 3 0 016 0zm6 2a9 9 0 11-18 0 9 9 0 0118 0z" />
+                                </svg>
+                                <span class="truncate">
+                                    Example API Response
+                                </span>
+                            </a>
+                        </nav>
+                    </aside>
+
+                    <div class="space-y-6 sm:px-6 lg:px-0 lg:col-span-9">
+                        <div id="cookies">
+                            <div class="shadow sm:rounded-md sm:overflow-hidden">
+                                <div class="bg-white">
+                                    <div class="px-4 py-5 sm:px-6 bg-gray-100">
+                                        <h3 class="text-lg leading-6 font-medium text-gray-900">Example API Response</h3>
+                                    </div>
+
+                                    <div class="px-4 py-5 sm:px-6 grid grid-cols-6 gap-6">
+                                        <div class="col-span-6">
+                                            <textarea id="backendApiResponse" readonly="readonly" class="mt-1 block w-full bg-white border border-gray-300 rounded-md shadow-sm py-2 px-3 focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"></textarea>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </main>
+
+        <script>
+            createAuth0Client({
+                domain: '<?php echo $config->getDomain(); ?>',
+                client_id: '<?php echo $config->getClientId(); ?>',
+                audience: '<?php echo $config->buildDefaultAudience(); ?>',
+                redirect_uri: '<?php echo $router->getUri('/', ''); ?>',
+                scope: '<?php echo $config->buildScopeString() ?>'
+            }).then(auth0 => {
+                document.getElementById('login').addEventListener('click', () => {
+                    auth0.loginWithRedirect().catch(() => {});
+                });
+
+                document.getElementById('logout').addEventListener('click', () => {
+                    auth0.logout();
```

Comment:
Should always specify `returnTo` here to save problems with a misconfigured Auth0 client (not specifying `returnTo` will use a global tenant value which I believe is not always defined, or wrong).

```js
auth0.logout({ returnTo: '<your app url>' });
```